cs/v1/CS_v1.0.0.yaml

```diff
@@ -397,15 +397,15 @@ paths:
             type: string
             minLength: 7
             maxLength: 8
-            pattern: ^\d{7,8}$
+            pattern: '^\d{7,8}$'
```

Comment:
Specially here the pattern is now wrong as the backlash is not escaped - I suggest to remove all quotes on patterns

Reply:
I will not comment on the rest of the quotes on patterns - it is the same problem